### PR TITLE
Lenny

### PR DESCRIPTION
**Pull request by leenderteloff**
please look at the buttonPress function case exitButton

### CODE REVIEW

**Comment by henrycity** [comment]
Resolve the conflict before making the PR. And if it is not complete, don't make a PR.

**Comment by danielcwilson** [comment]
Yeah since we dont know how to merge with conflicts in SourceTree or in the shell this was the easiest way to do it

File: MobileDevProject/ViewController.swift [hunk 1]
```diff
@@ -12,19 +12,26 @@ import UIKit
 import XLActionController
 
 class ViewController: UIViewController, CLLocationManagerDelegate {
-
+
```
Comment (henrycity) [inline MobileDevProject/ViewController.swift:15]: Don't make unnecessary changes!

Comment (henrycity) [reply MobileDevProject/ViewController.swift:15]: Check the linter before commiting.

File: MobileDevProject/ViewController.swift [hunk 2]
```diff
@@ -205,7 +212,32 @@ class ViewController: UIViewController, CLLocationManagerDelegate {
                         }
                 ))
                 present(actionController, animated: true, completion: nil)
-            case beaconButton3:
+            case beaconButton3: break
+<<<<<<< .merge_file_iRmVcJ
```
Comment (henrycity) [inline MobileDevProject/ViewController.swift:216]: what is this?

File: MobileDevProject/ViewController.swift [hunk 3]
```diff
@@ -220,6 +252,7 @@ class ViewController: UIViewController, CLLocationManagerDelegate {
             default:
                 print("Unknown button")
                 return
+>>>>>>> .merge_file_hMsxl8
```
Comment (henrycity) [inline MobileDevProject/ViewController.swift:255]: again?

File: MobileDevProject/ViewController.swift [hunk 4]
```diff
@@ -205,7 +212,32 @@ class ViewController: UIViewController, CLLocationManagerDelegate {
...
+                print("Unknown button")
+                return
+            }
+        
+    
+            addLine(fromPoint: nearestBeacon, toPoint: selectedBeacon)
+            updateCircle(fromPoint: nearestBeacon, toPoint: selectedBeacon)
+            navigatingBeacon = selectedBeacon
+            isNavigating = true
+=======
```
Comment (henrycity) [inline MobileDevProject/ViewController.swift:240]: ?

**Review by henrycity — CHANGES_REQUESTED** [review]

File: MobileDevProject/ViewController.swift (hunk 3 above)
Comment (leenderteloff) [reply MobileDevProject/ViewController.swift:255]: wtf... All the things you comment on are not in my project..

Comment (danielcwilson) [reply MobileDevProject/ViewController.swift:255]: Possibly an issue with the webmerge I did, try pull and see if theres a change

**Pull request closed without merging**